### PR TITLE
cross-signing UI fixes

### PR DESCRIPTION
**Pull request by gtalha07**
Some changes needed in cross-signing UI that came from #177.

### CODE REVIEW

File: app/lib/common/widget/CrossSigning.dart [hunk 1]
```diff
@@ -215,30 +215,81 @@ class CrossSigning {
...
+                        color: Colors.white,
+                      ),
+                    ),
+                  ],
                 ),
-                Text(
-                  AppLocalizations.of(context)!.verificationScanSelfNotice,
-                  style: CrossSigningSheetTheme.secondaryTextStyle,
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(15.0, 8.0, 15, 8.0),
```
Comment (bitfriend) [inline app/lib/common/widget/CrossSigning.dart:247]: You may change like this snippet.
`const EdgeInsets.symmetric(horizontal: 15.0, vertical: 8.0)`

File: app/lib/l10n/app_en.arb [hunk 2]
```diff
@@ -88,6 +88,7 @@
     "verificationSasDoNotMatch":"They don’t match",
     "verificationEmojiNotice":"Compare the unique emoji, ensuring they appear in the same order.",
     "verificationCodeNotice":"Compare the code with the one displayed on the other user’s screen.",
+    "verificationConclusionOkTitle":"Verified",
```
Comment (bitfriend) [inline app/lib/l10n/app_en.arb:91]: Already `sasVerified` is defined.

**Review by Cervantes12 — APPROVED** [review]

File: app/lib/l10n/app_en.arb (hunk 2 above)
Comment (gtalha07) [reply app/lib/l10n/app_en.arb:91]: My bad. Didn't looked closely.

**Comment by sachintha-s** [comment]
@gtalha07 I tried several times to build this PR but every time I got this screen.
image

**Review by sachintha-s — COMMENTED** [review]
@gtalha07 I tried several times to build this PR but every time I got this screen.
image

**Comment by bitfriend** [comment]
@sachintha-s Your app loaded `libeffektio.so`?
There is no error like this on your side?
image

**Comment by sachintha-s** [comment]
@bitfriend  There is an exception like this.
image

**Comment by bitfriend** [comment]
Yes, it seems to not have correct rust library.

File: app/lib/common/widget/CrossSigning.dart [hunk 3]
```diff
@@ -685,22 +808,10 @@ class CrossSigning {
                     const SizedBox(width: 5.0),
                     Text(
                       _eventMap[txnId]!.verifyingThisDev
```
Comment (bitfriend) [inline app/lib/common/widget/CrossSigning.dart:810]: You may remove condition here, because you used only single text.

**Review by Cervantes12 — APPROVED** [review]

**Review by sachintha-s — APPROVED** [review]

**Pull request merged**